### PR TITLE
Add ConfirmationModal.tsx file

### PR DESCRIPTION
**Pull request by hacheG**

### CODE REVIEW

File: src/components/ConfirmationModal.jsx [hunk 1]
```diff
@@ -1,49 +0,0 @@
...
-  styleCustom: PropTypes.string,
-  onContinue: PropTypes.func.isRequired,
-  onCancel: PropTypes.func.isRequired,
-};
-
-ConfirmationModal.defaultProps = {
-  open: false,
-  onClose: null,
-  styleCustom: null,
-};
```
Comment (cazapatamar) [inline src/components/ConfirmationModal.jsx:47]: By passing to TS, you are missing the required (mandatory) props and default values

Comment (hacheG) [reply src/components/ConfirmationModal.jsx:47]: ok, I changed the props: open to false and styleCustom to null, but the onClose function cannot be sent to null by default. On the other hand, the interface values are required by not having the symbol: ?

Comment (cazapatamar) [reply src/components/ConfirmationModal.jsx:47]: Try using an empty function ()=>{} and read about whether it is possible to use it to replace the null value for onClose function.

Yes, ? is used to make an attribute non-mandatory in the interface. I understand that if you don't use ?, the attribute is mandatory.

Comment (hacheG) [reply src/components/ConfirmationModal.jsx:47]: it's so strange, I changed to () => {} and the Uim.tsx shows me an error ( 102 onClose= ), but biotablero doesn't crash

Comment (cazapatamar) [reply src/components/ConfirmationModal.jsx:47]: I just make the change using () => {} as a default value for onClose and it works for me

**Review by cazapatamar — APPROVED** [review]

**Pull request merged**